lab_2_keywords_cooccurrence/main.py

```diff
@@ -17,7 +17,34 @@ def extract_phrases(text: str) -> Optional[Sequence[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(text, str):
+        return None
+    if text == '':
+        return None
+    if all(symbol.isalpha() is False for symbol in text):
+        return []
+    level_one = text[:]
+    for symbol in level_one:
+        if symbol in ['!', '(', ')', '-', '[', ']', '{', '}', ';', ':', '"', ',', "'", '¡', '¿', '-',
```

Comment:
move the punctuation a variable. Or use `string.punctuation`

Reply:
done